### PR TITLE
{bio}[gompi/2020a] Modeltest-NG v0.1.7

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/077ac180b13a24cec75fc32eb4cb24ad for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/092a7e444df38e2b6f0ec145a11b627d for a full test report.

File: easybuild/easyconfigs/m/Modeltest-NG/Modeltest-NG-0.1.7-gompi-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        'url': 'https://github.com/ddarriba',
+        'repo_name': 'modeltest',
+        'tag': 'v%(version)s',
+        'recursive': True,
+    },
+}]
+
+checksums = [None]
+
+dependencies = [
```
Comment (verdurin) [inline easybuild/easyconfigs/m/Modeltest-NG/Modeltest-NG-0.1.7-gompi-2020a.eb:25]: @FokkeDijkstra I would remove `dependencies` if there aren't any.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/Modeltest-NG/Modeltest-NG-0.1.7-gompi-2020a.eb (hunk 1 above)
Comment (FokkeDijkstra) [reply easybuild/easyconfigs/m/Modeltest-NG/Modeltest-NG-0.1.7-gompi-2020a.eb]: Done.

**Comment by boegel** [comment]
closing this since `foss/2020a` is deprecated, see https://docs.easybuild.io/policies/toolchains

Sorry for not getting back to this @FokkeDijkstra 

If `Modeltest-NG` is still relevant, please consider opening a new pull request using a more toolchain

**Pull request closed without merging**